logs.go

```diff
@@ -0,0 +1,33 @@
+package empire
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/docker/docker/pkg/jsonmessage"
+	streamhttp "github.com/remind101/empire/pkg/stream/http"
+	"github.com/remind101/kinesumer"
+)
+
+func (e *Empire) StreamLogs(a *App, w http.ResponseWriter) error {
```

Comment:
Also, this is the only place where we add methods onto the Empire struct outside of `empire.go`. I think it would make sense to have a `logsService` interface, where the default implementation is basically just a null implementation, since not everyone is going to want to use kinesis.

Maybe something like:

```go
// logs.go

type logsService interface {
        StreamLogs(context.Context, *App, io.Writer) error
}

type nullLogsService struct{}

func (s *nullLogsService) StreamLogs(ctx context.Context, app *App, w io.Writer) error {
        io.WriteString(w, "Logs are disabled")
        return nil
}

type kinesisLogsService struct {
}

func (s *kinesisLogsService) StreamLogs(ctx context.Context, app *App, w io.Writer) error {
        // ....
}
```

```go
// empire.go

func (e *Empire) StreamLogs(ctx context.Context, app *App, w io.Writer) error {
        return e.logsService.StreamLogs(ctx, app, w)
}
```

And then maybe a CLI flag to enable kinesis streams.